requirements/test.txt

```diff
@@ -1,4 +1,5 @@
 case>=1.3.1
-pytest>=3.0
 pytest-django
 pytz>dev
+pytest<4.0.0
```

Comment:
we should use latest pytest as celery also use that.

Reply:
I was getting errors from this with 4.0+

```
AttributeError: 'TestCaseFunction' object has no attribute 'get_marker'
```

Reply:
i will push pytest update later. keep this less then 4 for now.